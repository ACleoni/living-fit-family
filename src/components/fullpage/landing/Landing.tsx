import React from 'react';
import styles from './Landing.module.scss';
import { Video } from 'cloudinary-react';

export default function Landing() {
  const isAdminMode: boolean = false;

  return (
    <section>
      <div className={styles.section}>
        <div className={styles.container}>
          <div className={styles.heroFilter} />
          <div className={styles.heroContainer}>
            <Video
              id={styles.video}
              cloudName={'dq0fvempx'}
              muted
              autoPlay
              loop
              playsInline
              data-keepplaying
              publicId='landing-page-video'
            ></Video>
          </div>
          {/* <i id={styles.moveDown} className='fa fa-chevron-down fa-3x bounce'></i> */}
        </div>
        {/* <div class='opaque-bg animated fadeInDown'>
          <h1 style='color:white'>
            BRAD<span style='color:#FF6363'>/</span>ENGELHARDT
          </h1>
          <p>
            <span id='holder'></span>
            <span class='blinking-cursor'>|</span>
          </p>
        </div> */}
      </div>
      <footer>
        <div>
          <p className={styles.footer}>STREAM FROM YOU PHONE OR LAPTOP WITH ZOOM</p>
          <i className='fa fa-mobile'></i>
          <i className='fa fa-laptop'></i>
        </div>
      </footer>
    </section>
  );
}
