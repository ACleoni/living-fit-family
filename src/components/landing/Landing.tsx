import React from 'react';
import { Box, Button, Typography } from '@material-ui/core';
import { Video } from 'cloudinary-react';

export default function Landing() {
  return (
    <div style={{ height: '60vh', position: 'relative' }}>
      <div
        style={{
          bottom: '0',
          left: '0',
          position: 'absolute',
          right: '0',
          top: '0',
          overflow: 'hidden',
          opacity: 0.3,
          background: '#000',
          zIndex: 100,
        }}
      ></div>
      <div style={{ height: '100%', position: 'relative' }}>
        <div style={{ position: 'absolute', bottom: '18px', left: '25px', zIndex: 200 }}>
          <Typography style={{ zIndex: 120 }} variant='h4' align='justify'>
            Take Living Fit To The
          </Typography>
          <Typography style={{ zIndex: 120, marginBottom: '16px' }} variant='h4' align='justify'>
            Next Level
          </Typography>
          {/* <div style={{ margin: '14px' }}></div> */}
          <Button color='primary' variant='contained' disableElevation>
            Start Training Online Now
          </Button>
        </div>
        <Video
          style={{
            width: '100%',
            height: '100%',
            objectFit: 'cover',
          }}
          cloudName={'dq0fvempx'}
          muted
          autoPlay
          loop
          playsInline
          data-keepplaying
          publicId='landing-page-video'
        ></Video>
      </div>
    </div>
  );
}
