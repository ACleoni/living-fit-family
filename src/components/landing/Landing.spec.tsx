import React from 'react';
import renderer from 'react-test-renderer';
import Landing from './Landing';

it('renders correctly', () => {
    const tree = renderer
        .create(<Landing></Landing>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});