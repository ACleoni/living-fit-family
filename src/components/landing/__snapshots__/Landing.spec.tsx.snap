// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<video
  autoPlay={true}
  data-keepplaying={true}
  id="#video"
  loop={true}
  muted={true}
  playsInline={true}
  poster="http://res.cloudinary.com/dq0fvempx/video/upload/landing-page-video.jpg"
>
  <source
    src="http://res.cloudinary.com/dq0fvempx/video/upload/landing-page-video.webm"
    type="video/webm"
  />
  <source
    src="http://res.cloudinary.com/dq0fvempx/video/upload/landing-page-video.mp4"
    type="video/mp4"
  />
  <source
    src="http://res.cloudinary.com/dq0fvempx/video/upload/landing-page-video.ogv"
    type="video/ogg"
  />
</video>
`;
